dadguide/dadguide.py

```diff
@@ -76,7 +76,8 @@ def __init__(self, bot, *args, **kwargs):
 
         self.fm_flags_default = {'na_prio': True, 'server': DEFAULT_SERVER}
         self.config = Config.get_conf(self, identifier=64667103)
-        self.config.register_global(datafile='', indexlog=0, test_suite={}, fluff_suite=[], typo_mods=[])
+        self.config.register_global(datafile='', indexlog=0, test_suite={}, fluff_suite=[], typo_mods=[],
+                                    ts_only=False, ts_only_mons = [3260])
```

Comment:
let's call it debug_mode=False
debug_mode_monsters=[]

and not refer to it as tsubaki_mode anywhere in the code except as an alias to the command